### ISSUE TITLE
Testing Individual Subject Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4632]
# Page to Test:

Subjects - https://openlibrary.org/subjects/architecture

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this.

**Comment by Yashs911** [comment]
@bpmcneilly There is no alt attribute for book covers on this page as well as all other pages.
alt for lists present in `widget-lists-wrapper` is broken
![image](https://user-images.githubusercontent.com/64412143/108587438-884e4500-7379-11eb-9988-60f1602a7988.png)

**Comment by bpmcneilly** [comment]
@Yashs911 I think the solution for the list images here will be similar to what we were discussing in #4633 - though it looks like the CSS might be slightly more challenging here.

for the carousels, it looks like we're gonna need the `title` attributes to become `alt`.
